WindowsAppSDK.Build.Cpp.props

```diff
@@ -0,0 +1,84 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Build with the hybrid CRT (Universal CRT + Static VS CRT (for what little the Universal CRT doesn't cover) -->
+  <Import Project="$(MSBuildThisFileDirectory)HybridCRT.props" />
+
+  <!-- Build with APIscan-friendly compiler+linker options -->
+  <Import Project="$(MSBuildThisFileDirectory)ApiScan.Cpp.props" />
+
+  <!-- Common (all Configurations) options -->
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <!-- /std:c++17 Enable ISO C++17 Standard -->
+      <LanguageStandard>stdcpp17</LanguageStandard>
+
+      <!-- /permissive- Enable Conformance mode (i.e. Disable 'permissive' mode) -->
+      <ConformanceMode>true</ConformanceMode>
+
+      <!-- /W4 Enable max warning level -->
+      <WarningLevel>Level4</WarningLevel>
```

Comment:
>WarningLevel [](http://example.com/codeflow?start=7&length=12)

What about /WX?

Reply:
Not yet. There's a handful of warnings (though you have to Rebuild to see some) but we're into the complex-weird territory (props/targets related, not source code).

/WX is expected in the foreseeable future